content/vendor/cli/getting-started.md

```diff
@@ -25,7 +25,7 @@ You’ll need to set up two environment variables to interact with https://vendo
 * `REPLICATED_APP` should be set to the name of your KOTS application, as shown in the URL path at https://vendor.replicated.com/apps. _Note that this is **case sensitive**_
 ![Vendor Application Slug](/images/vendor-app-slug.png)
 
-* `REPLICATED_API_TOKEN` should be set to a token created at https://vendor.replicated.com/team/tokens.
+* `REPLICATED_API_TOKEN` should be set to a team token created at [https://vendor.replicated.com/team/tokens](https://vendor.replicated.com/team/tokens), or a user token created at https://vendor.replicated.com/account-settings.
```

Comment:
@jeffreygolden I don't know if this is the most succinct way to do it but should work :D